doc/nvim-tree-lua.txt

```diff
@@ -1483,121 +1600,6 @@ Examples:
 
           ----
 <
-Mouse support defined in |KeyBindings|
-
-Single left mouse mappings can be achieved via `<LeftRelease>`.
-
-Single right / middle mouse mappings will requre changes to |mousemodel| or |mouse|.
-
-DEFAULT MAPPINGS                                     *nvim-tree-default-mappings*
```

Comment:
Removed these as they are problematic to generate; DEFAULT_ON_ATTACH is quite readable.